tests/unit/iontests.js

```diff
@@ -273,46 +273,43 @@ define([
             return fs.readFileSync(path, options);
         }
 
+        function makeStream(src, makeWriter) {
+            let writer = makeWriter;
```

Comment:
In my feedback, I suggested that `makeWriter` be a function argument, not the `writer` itself.

If you make this actually a *function* (i.e. callable), you can streamline the code below as:

```javascript
streams.push(makeStream(input, ion.makeTextWriter));
```

And so-on.  If you choose not to take this advice, please don't have a redundant parameter called `makeWriter` that is literally just assigned to a local named `writer`.

Reply:
I went with streams.push(makeStream(input, ion.makeTextWriter()));